lib/rules/no-unused-class.js

```diff
@@ -107,14 +109,14 @@ export default {
            ```
            then the loop will be run 2 times
          */
-        _.forOwn(map, (o) => {
+        _.forIn(map, (o) => {
```

Comment:
Switched to forIn for performance reasons. No one modifies Object.prototype anymore :D